test/e2e/mist_config.go

```diff
@@ -105,6 +108,7 @@ func defaultMistConfig() mistConfig {
 				{Connector: "TSSRT"},
 				{Connector: "WAV"},
 				{Connector: "WebRTC"},
+				{Connector: "livepeer-catalyst-node", RPCAddr: fmt.Sprintf("0.0.0.0:%s", serfPort)},
```

Comment:
@hjpotter92 Can you figure out how to exempt `test/**` from code coverage tests? It's already a test.

<img width="531" alt="image" src="https://user-images.githubusercontent.com/257909/182261367-d2eaf529-a0b5-4748-a9ff-ca0e1ecfd3a6.png">